include/emqx_ct_helpers.hrl

```diff
@@ -0,0 +1,140 @@
+					"TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
+					"TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384",
+					"TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384",
+					"TLS_ECDH_ECDSA_WITH_AES_256_GCM_SHA384",
+					"TLS_ECDH_RSA_WITH_AES_256_GCM_SHA384",
+					"TLS_ECDH_ECDSA_WITH_AES_256_CBC_SHA384",
+					"TLS_ECDH_RSA_WITH_AES_256_CBC_SHA384",
+					"TLS_DHE_RSA_WITH_AES_256_GCM_SHA384",
+					"TLS_DHE_DSS_WITH_AES_256_GCM_SHA384",
+					"TLS_DHE_RSA_WITH_AES_256_CBC_SHA256",
+					"TLS_DHE_DSS_WITH_AES_256_CBC_SHA256",
+					"TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256",
+					"TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
+					"TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256",
+					"TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256",
+					"TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256",
+					"TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256",
+					"TLS_ECDH_ECDSA_WITH_AES_128_GCM_SHA256",
+					"TLS_ECDH_RSA_WITH_AES_128_GCM_SHA256",
+					"TLS_ECDH_ECDSA_WITH_AES_128_CBC_SHA256",
+					"TLS_ECDH_RSA_WITH_AES_128_CBC_SHA256",
+					"TLS_DHE_RSA_WITH_AES_128_GCM_SHA256",
+					"TLS_DHE_DSS_WITH_AES_128_GCM_SHA256",
+					"TLS_DHE_RSA_WITH_CHACHA20_POLY1305_SHA256",
+					"TLS_DHE_RSA_WITH_AES_128_CBC_SHA256",
+					"TLS_DHE_DSS_WITH_AES_128_CBC_SHA256",
+					"TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA",
+					"TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA",
+					"TLS_DHE_RSA_WITH_AES_256_CBC_SHA",
+					"TLS_DHE_DSS_WITH_AES_256_CBC_SHA",
+					"TLS_ECDH_ECDSA_WITH_AES_256_CBC_SHA",
+					"TLS_ECDH_RSA_WITH_AES_256_CBC_SHA",
+					"TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA",
+					"TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA",
+					"TLS_DHE_RSA_WITH_AES_128_CBC_SHA",
+					"TLS_DHE_DSS_WITH_AES_128_CBC_SHA",
+					"TLS_ECDH_ECDSA_WITH_AES_128_CBC_SHA",
+					"TLS_ECDH_RSA_WITH_AES_128_CBC_SHA",
+					"TLS_SRP_RSA_WITH_3DES_EDE_CBC_SHA",
+					"TLS_SRP_DSS_WITH_3DES_EDE_CBC_SHA",
+					"TLS_SRP_RSA_WITH_AES_128_CBC_SHA",
+					"TLS_SRP_DSS_WITH_AES_128_CBC_SHA",
+					"TLS_SRP_RSA_WITH_AES_256_CBC_SHA",
+					"TLS_SRP_DSS_WITH_AES_256_CBC_SHA",
+					"TLS_RSA_WITH_AES_256_GCM_SHA384",
+					"TLS_RSA_WITH_AES_256_CBC_SHA256",
+					"TLS_RSA_WITH_AES_128_GCM_SHA256",
+					"TLS_RSA_WITH_AES_128_CBC_SHA256",
+					"TLS_RSA_WITH_AES_256_CBC_SHA",
+					"TLS_RSA_WITH_AES_128_CBC_SHA",
+					"TLS_RSA_WITH_3DES_EDE_CBC_SHA",
+					"TLS_DHE_RSA_WITH_DES_CBC_SHA",
+					"TLS_RSA_WITH_DES_CBC_SHA",
+					"TLS_ECDHE_ECDSA_WITH_3DES_EDE_CBC_SHA",
+					"TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA",
+					"TLS_DHE_RSA_WITH_3DES_EDE_CBC_SHA",
+					"TLS_DHE_DSS_WITH_3DES_EDE_CBC_SHA",
+					"TLS_ECDH_ECDSA_WITH_3DES_EDE_CBC_SHA",
+					"TLS_ECDH_RSA_WITH_3DES_EDE_CBC_SHA",
+					"TLS_ECDHE_ECDSA_WITH_RC4_128_SHA",
+					"TLS_ECDHE_RSA_WITH_RC4_128_SHA",
+					"TLS_ECDH_ECDSA_WITH_RC4_128_SHA",
+					"TLS_ECDH_RSA_WITH_RC4_128_SHA",
+					"TLS_RSA_WITH_RC4_128_SHA",
+					"TLS_RSA_WITH_RC4_128_MD5"
+			    ]}
+	]).
+
+-define(TLS_OLD_CIPHERS,
+	[ { versions, [ 'tlsv1.1', 'tlsv1.2' ] },
+	  { ciphers,  [ "ECDHE-ECDSA-AES256-GCM-SHA384",
```

Comment:
ssl:cipher_suites(exclusive, 'tlsv1.2', openssl) ++ ssl:cipher_suites(exclusive, 'tlsv1.1', openssl).

Reply:
Explicit use of the strings to match what would be typed in config, as this is a Test Suite helper.